nxdrive/options.py

```diff
@@ -157,6 +157,16 @@ def __call__(self, new_value: bool, /) -> None:
             setattr(Feature, self._feature, new_value)
 
 
+class SynchronzationHandler(CallableFeatureHandler):
+    def __init__(self, feature: str, /) -> None:
+        super().__init__(feature)
+
+    def __call__(self, new_value: bool, /) -> None:
+        if getattr(Feature, self._feature) is not new_value:
+            setattr(Feature, self._feature, new_value)
+            Options.synchronization_enabled = new_value
```

Comment:
I would prefer something that does not repeat what the parent's method does. Improve the parent's implementation to be able to do:
```suggestion
    def __call__(self, new_value: bool, /) -> None:
        if super().__call__(new_value):
            Options.synchronization_enabled = new_value
```